apps/website/contents/resume.md

```diff
@@ -343,6 +343,66 @@ Simply copy the content from your resume and paste it into a plain text document
 - Characters are displaying incorrectly in plain text
 - Sections are disorganized
 
+## Crafting the Perfect Cover Letter
+
+Your cover letter serves as a handshake, an introduction to the professional persona encapsulated in your resume. It's an avenue to express your genuine interest in a position or a particular company, portraying the value you can add to the organization. Unlike the formal tone of a resume, a cover letter allows for a unique, creative expression of your personal brand, ensuring it **complements rather than replicates** your CV.
```

Comment:
Can we be consistent and use "Resume" instead of "CV" throughout?

Reply:
✅ Fixed. Changed all the instances from "CV" to "resume".